test/SILOptimizer/mandatory_inlining.swift

```diff
@@ -187,3 +187,19 @@ func dontCrash() {
     fatalError("baz \(k)")
   }
 }
+
+func switchLoopWithPartialApplyCallee(reportError: ((String) -> (Void))?) {
```

Comment:
For anyone reading this, the code has an assert in it that would trigger on this.